test/cpp/qps/client_async.cc

```diff
@@ -949,10 +949,125 @@ std::unique_ptr<Client> CreateAsyncClient(const ClientConfig& config) {
       return nullptr;
   }
 }
+
 std::unique_ptr<Client> CreateGenericAsyncStreamingClient(
     const ClientConfig& args) {
   return std::unique_ptr<Client>(new GenericAsyncStreamingClient(args));
 }
 
+class CallbackClient
+    : public ClientImpl<BenchmarkService::Stub, SimpleRequest> {
+ public:
+  CallbackClient(const ClientConfig& config)
+      : ClientImpl<BenchmarkService::Stub, SimpleRequest>(
+            config, BenchmarkStubCreator) {
+    num_threads_ =
+        config.outstanding_rpcs_per_channel() * config.client_channels();
```

Comment:
Why have so many threads for an async benchmark? The callback RPCs are nonblocking, so can't we just get by with far fewer threads (just like in the existing async QPS test)?